src/modules/ttlmonitor/geometry/Parser.h

```diff
@@ -62,6 +64,6 @@ class Parser {
   struct ParserStats Stats;
 
 private:
-  const uint16_t DataLength{16};
+  const uint16_t DataLength{20};
```

Comment:
Shall we use DATASIZE here instead hardcoded 20?

Reply:
yes, will do.